app/Console/Commands/SyncMedia.php

```diff
@@ -63,24 +63,26 @@ public function logToConsole($path, $result)
     {
         $name = basename($path);
 
-        if ($result === true) {
+        if (is_null($result)) {
```

Comment:
[Not really](https://github.com/phanan/koel/blob/master/app/Services/Media.php#L89).